src/Api/ReportApi.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Silktide\ProspectClient\Api;
+
+use Silktide\ProspectClient\ApiRequest\CreateReportApiRequest;
+use Silktide\ProspectClient\ApiRequest\FetchReportApiRequest;
+use Silktide\ProspectClient\ApiRequest\ReanalyzeReportApiRequest;
+use Silktide\ProspectClient\ApiRequest\SearchReportApiRequest;
+use Silktide\ProspectClient\Entity\Report;
+
+class ReportApi extends AbstractApi
+{
+    public function create(): CreateReportApiRequest
```

Comment:
The documentation currently gives an example of feeding a URL to this create function, but this function does not accept it.

It's worth noting that `create` can be done using just business details, so if there is a url parameter here, it should be an optional requirement